packages/identity-service/src/authMiddleware.js

```diff
@@ -49,6 +97,7 @@ async function authMiddleware(req, res, next) {
     const encodedDataMessage = req.get('Encoded-Data-Message')
     const signature = req.get('Encoded-Data-Signature')
     const handle = req.query.handle
+    const actingUserId = req.query.actingAsUserId
```

Comment:
As in `?actingAsUserId=12345`? It does deviate from the pattern we use everywhere else of just passing `user_id=`

Reply:
Ill change this to just user_id 👍